release.sh

```
@@ -0,0 +1,316 @@
+    grep '^version = ' "$CARGO_TOML" | head -1 | sed 's/version = "\(.*\)"/\1/'
+}
+
+set_version() {
+    local new="$1"
+    sed -i '' "s/^version = \".*\"/version = \"${new}\"/" "$CARGO_TOML"
+    echo "Version set to $new"
+}
+
+# Strip any suffix (-dev, -beta.N, etc.) → pure semver
+strip_suffix() {
+    echo "$1" | sed 's/-.*//'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)